### ISSUE TITLE
Cannot copy elements from one map to another using STL

### ISSUE THREAD

**Issue by h-2** [martinus/robin-hood-hashing#139]
Happy new year and thanks for providing this library first of all!

The following fails to build:
```cpp
    robin_hood::unordered_flat_map<int, int> in;
    robin_hood::unordered_flat_map<int, int> out;

    std::copy(in.begin(), in.end(),
              std::inserter(out, out.begin()));
```

Error message at godbolt: https://godbolt.org/z/r51z49d5x

**Comment by h-2** [comment]
The problem here is that your insert functions return `std::pair<iterator, bool>` while the insert functions in `std::unordered_map` and `phmap::flat_hash_map` etc return just `iterator`.

**Comment by slavenf** [comment]
@h-2 Problem is resolved in version 3.11.5.

This issue can be closed.
